bitnami/grafana/values.yaml

```diff
@@ -307,6 +308,7 @@ livenessProbe:
 ##
 readinessProbe:
   enabled: true
+  scheme: HTTP
```

Comment:
Hi,
  You need to add this parameter to the comment block above. This will help to autogenerate parameters sections in the README.md file.

Reply:
Same as the above, but for readinessProbe:
```
  ## @param readinessProbe.scheme Scheme for readinessProbe
```